packages/rx_bloc_cli/mason_templates/bricks/feature_otp/__brick__/lib/feature_otp/lib_sms_code_verification/ui_components/resend_code_button.dart

```diff
@@ -0,0 +1,221 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/widget_toolkit.dart';
+
+import '../../../app_extensions.dart';
+import '../../../base/models/temporary_code_state.dart';
+import '../bloc/sms_code_bloc.dart';
+
+//It builds a TextButton with all necessary functionalities and state handling
+// with great amount of customization options and automated styling.
+class ResendCodeButton extends StatelessWidget {
+  const ResendCodeButton(
```

Comment:
It looks like the files from the widget_toolkit_otp package are copied inside the feature_otp brick folder, instead of just using them from the package.